weave/trace_server/calls_query_builder/calls_query_builder.py

```diff
@@ -763,7 +763,7 @@ def _as_sql_base_format(
             pb,
             table_alias,
         )
-        sortable_datetime_sql = (
+        sortable_datetime_opt_sql = (
```

Comment:
why are you renaming these?

Reply:
Now, you can _easily_ tell which of the filters are optimizations, which require an additional condition after the `HAVING` block, and which **replace** the having block entirely.